.circleci/config.yml

```diff
@@ -203,6 +216,15 @@ jobs:
     docker:
       - image: node
 
+  node-bench-e2e-latest:
+    <<: *node-bench-e2e-base
+    docker:
+      - image: node
+        environment:
+          - SERVICES=mongo
+          - PLUGINS=mongodb-core
```

Comment:
Yeah we need that library to test connectivity before running the app. It makes sense to do it using the `versions` version of it rather than trusting the app's install to have it in an expected place.